src/main/resources/assets/forestry/lang/en_us.lang

```diff
@@ -1804,7 +1794,7 @@ for.config.tweaks.book.spawn.comment=Players who enter the world for the first t
 for.config.genetics=Genetics
 for.config.genetics.pollinate.vanilla.trees=Pollinate Vanilla Trees
 for.config.genetics.pollinate.vanilla.trees.comment=Allow bees to pollinate vanilla tree leaves. When disabled, vanilla trees must be analyzed before they can be pollinated.
-for.config.genetics.analyzerblock.energy.use=Energy use of Analyzer Block per work cycle.
+for.config.genetics.analyzerblock.energy.use.comment=Energy use of Analyzer Block per work cycle.
```

Comment:
Could you please add a name for this config option too?

Reply:
Woops, fixed

Reply:
Thanks